lib/load/ws_handler.ex

```diff
@@ -55,15 +69,18 @@ defmodule Load.WSHandler do
     {:reply, {:text, Jason.encode!(%{stats: stats})}, state}
   end
 
-  @impl true
+  def websocket_info(:ping, state) do 
+    {:ok, state}
+  end
```

Comment:
should be the server sending pings and receiving back pongs, should we remove this?